etna/records/models.py

```diff
@@ -182,12 +182,20 @@ def hierarchy_level3_summary_title(self) -> str:
     @cached_property
     def next_record(self) -> Union[Dict[str, str], None]:
         if record := self.get("@next"):
-            return {"iaid": extract(record, "@admin.id")}
+            ref_num = self.get("identifier.0.reference_number")
+            return dict(
+                iaid=extract(record, "@admin.id"),
+                reference_number=ref_num,
+            )
 
     @cached_property
     def previous_record(self) -> Union[Dict[str, str], None]:
         if record := self.get("@previous"):
-            return {"iaid": extract(record, "@admin.id")}
+            ref_num = self.get("identifier.0.reference_number")
```

Comment:
@jamesbiggs1 I don't think `self.get()` is the right thing to use here... that will give you the current record's reference number, when what you want is the one for the previous record.

Reply:
Hi Andy, cheers for pointing this out - I didn't even realise... I was having an issue referencing the @next.identifier, so I thought I did it right when it worked! I've fixed this now, cheers :)